core/src/main/java/com/rim/IceField/PlayerBase.java

```diff
@@ -169,8 +168,8 @@ public void die() {
     //Player falls into water
     public void fall() {
         currentIceberg.setType("hole");
+        isDrowning = true;
```

Comment:
theoretically, if the current iceberg is a hole, you don't need to set by yourself the isDrawing to true. If that is not working properly, please let us know and we would fix it.

Reply:
Actually the thing is that any player who is in the water is drawinig, but when isDrowning = true in the if it is only for those who don't have diving suits, that's not good. That's why each player in the water is drowning